ui/main.qml

```diff
@@ -187,6 +187,11 @@ StatusWindow {
                 Style.changeTheme(localAppSettings.theme, systemPalette.isCurrentSystemThemeDark())
                 Style.changeFontSize(localAccountSensitiveSettings.fontSize)
                 Theme.updateFontSize(localAccountSensitiveSettings.fontSize)
+            } else if(state === Constansts.appState.appEncryptionProcess) {
```

Comment:
Just wondering, does the backend signal the end of the encryption process somehow? Or, how does it transition back to the correct `Constants.appState.main` state?

Reply:
in the same way as was before :)